### ISSUE TITLE
[Feature Request] Create container image which have podman-remote binary for all platform and arch

### ISSUE THREAD

**Issue by praveenkumar** [containers/podman#14664]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**
As of now we release the podman-remote binary https://github.com/containers/podman/releases/tag/v4.1.1 it would be good if we have those binaries (not the msi) as part of single container image (something like podman-remote distribution image) which can be triggered during any PR on specific branch/tag or even master and always updated. This will help podman-remote consumer to not depend on the release bits if some bug introduced to it like https://github.com/containers/podman/issues/14201 one and fixed on main branch so able to consume the nightly container image.

This should be something similar what OpenShift client binary (oc) have as a single container image consist of all platform binaries.

**Comment by rhatdan** [comment]
If we had quay.io/podman/stable available for multi-arch, wouldn't this achieve the same thing, since you can execute

podman -remote from within these containers?

**Comment by praveenkumar** [comment]
@rhatdan yes having it as multi-arch helps. Executing `podman-remote` within containers need a container runtime but if a user just want platfrom (win/mac/linux) specific remote binary then they can download this image as tarball and extract or use skopeo to copy it local dir and get respective platform specific remote binary.

**Comment by rhatdan** [comment]
I would suggest you open a PR to create such an image.

**Comment by praveenkumar** [comment]
@rhatdan Sure, l will try to put a PR and see how it goes.

**Comment by rhatdan** [comment]
@praveenkumar were you able to work on this?

**Comment by praveenkumar** [comment]
@rhatdan Not yet :( , let me pick it up during this week (it is learn week).

**Comment by vrothberg** [comment]
Since https://github.com/containers/podman/pull/15813 merged, we can close
